compiler/cpp/src/thrift/generate/t_go_generator.cc

```diff
@@ -1493,11 +1493,15 @@ void t_go_generator::generate_go_struct_definition(ostream& out,
 
   if (is_exception) {
     out << indent() << "func (p *" << tstruct_name << ") Error() string {" << endl;
-    out << indent() << indent() << "return p.String()" << endl;
+    indent_up();
+    out << indent() << "return p.String()" << endl;
+    indent_down();
     out << indent() << "}" << endl << endl;
 
     out << indent() << "func (" << tstruct_name << ") TExceptionType() thrift.TExceptionType {" << endl;
-    out << indent() << indent() << "return thrift.TExceptionTypeCompiled" << endl;
+    indent_up();
+    out << indent() << "return thrift.TExceptionTypeCompiled" << endl;
+    indent_down();
```

Comment:
These are cosmetic fix of a wrongly did indent from my previous PR :$  not really related to this PR.